assembly/src/assembler/instruction/crypto_ops.rs

```diff
@@ -25,28 +25,39 @@ const RPHASH_NUM_ELEMENTS: u64 = 8;
 ///
 /// This operation takes 16 VM cycles.
 pub(super) fn rphash(span: &mut SpanBuilder) -> Result<Option<CodeBlock>, AssemblyError> {
-    // Add 4 elements to the stack to prepare the capacity portion for the Rescue Prime permutation
-    // The capacity should start at stack[8], and the number of elements to be hashed should
-    // be deepest in the stack at stack[11]
-    span.push_op(Push(Felt::new(RPHASH_NUM_ELEMENTS)));
-    span.push_op_many(Pad, 3);
-    span.push_op(SwapW2);
-    // restore the order of the top 2 words to be hashed
-    span.push_op(SwapW);
-
-    // Do the Rescue Prime permutation on the top 12 elements in the stack
-    span.push_op(RpPerm);
-
-    // Drop 4 elements (the part of the rate that doesn't have our result)
-    span.push_op_many(Drop, 4);
-
-    // Move the top word (our result) down the stack
-    span.push_op(SwapW);
-
-    // Drop 4 elements (the capacity portion)
-    span.push_op_many(Drop, 4);
-
-    Ok(None)
+    #[rustfmt::skip]
+    let ops = [
+        // Add 4 elements to the stack to prepare the capacity portion for the Rescue Prime permutation
+        // The capacity should start at stack[8], and the number of elements to be hashed should
+        // be deepest in the stack at stack[11]
+        Push(Felt::new(RPHASH_NUM_ELEMENTS)),
+        Pad,
+        Pad,
+        Pad,
+        SwapW2,
+
+        // restore the order of the top 2 words to be hashed
+        SwapW,
+
+        // Do the Rescue Prime permutation on the top 12 elements in the stack
+        RpPerm,
+
+        // Drop 4 elements (the part of the rate that doesn't have our result)
+        Drop,
+        Drop,
+        Drop,
+        Drop,
```

Comment:
minor nit: should these be listed in one row?

Reply:
Fixed